immersive-web-wg-charter.html

```diff
@@ -158,19 +160,16 @@ <h2>Scope</h2>
           that are embeddable in the Web of today, enabling progressive enhancement 
           of existing sites.
         </p>
+        <p>
+          A new generation of head-mounted displays and environment sensing capabilities on mobile devices are enabling augmented and virtual reality (collectively known as XR) to emerge as a critical field of evolution for human-machine interactions. Due to its inherent low friction and support for ephemeral experiences, the Web provides a promising ecosystem for the creation, distribution, and experiencing of XR content, applications, and services.
+        </p>
         <p>
           The <strong>scope</strong> of the Immersive Web Working Group charter is to
           define APIs which:
         </p>
         <ul>
-          <li>Detect available XR devices and sensors.
-          </li>
-          <li>Query XR devices for device-specific capabilities.
-          </li>
-          <li>Receive updated information about the device's position and orientation over time.
-          </li>
-          <li>Receive updated information about the device's environment.
-          </li>
+          <li>Detect the UAs capability to render XR scenes and report XR sensor values, including the user’s desired viewing position and orientation over time</li>
```

Comment:
Should be "UA's".

Oh, and on re-reading - should be "the device's position and orientation over time", not "the user's desired viewing position and orientation over time".  (It's a real measurement, not a virtual suggestion.)

Reply:
thank you for catching.
also added end dot for two lines.